benchmarks/rf_use_case/run_benchmark.R

```diff
@@ -0,0 +1,105 @@
+library(mlr3)
+library(data.table)
+library(mlr3torch)
+library(paradox)
+
+library(here)
+
+# define the tasks
+cc18_small = fread(here("data", "oml", "cc18_small.csv"))
+cc18_small_datasets = mlr3misc::pmap(cc18_small, function(data_id, name, NumberOfFeatures, NumberOfInstances) {
+  dt_name = here("data", "oml", paste0(name, "_", data_id, ".csv"))
+  fread(dt_name)
+})
+# cc18_small_datasets
+
+# cc18_small_datasets[[1]]
+
+# TODO: determine whether we can use OML tasks "directly"
+# didn't do this at first because they come with resamplings and we want to use our own resamplings
+kc1_1067 = as_task_classif(cc18_small_datasets[[1]], target = "defects")
+blood_1464 = as_task_classif(cc18_small_datasets[[2]], target = "Class")
+
+tasks = list(kc1_1067, blood_1464)
+
+# define the learners
+mlp = lrn("classif.mlp",
+  activation = nn_relu,
+  neurons = to_tune(
```

Comment:
```suggestion
  neurons = to_tune(ps(n_layers = p_int
```

Reply:
I think this won't work because the libraries don't allow parameter transformations. When I try to run the experiment I get this error:

`Error: Inner tuning and parameter transformations are currently not supported.`

Reply:
My solution for now:

`n_layers_values <- 1:10
latent_dim_values <- seq(10, 500, by = 10)
neurons_search_space <- mapply(
  neurons,
  expand.grid(n_layers = n_layers_values, latent_dim = latent_dim_values)$n_layers,
  expand.grid(n_layers = n_layers_values, latent_dim = latent_dim_values)$latent_dim,
  SIMPLIFY = FALSE
)

mlp = lrn("classif.mlp",
  activation = nn_relu,
  # neurons = to_tune(ps(
  #   n_layers = p_int(lower = 1, upper = 10), latent = p_int(10, 500),
  #   .extra_trafo = function(x, param_set) {
  #     list(neurons = rep(x$latent, x$n_layers))
  #   })
  # ),
  neurons = to_tune(neurons_search_space)`